libraries/AP_Compass/AP_Compass_RM3100.h

```diff
@@ -23,7 +23,10 @@
 #include "AP_Compass_Backend.h"
 
 #ifndef HAL_COMPASS_RM3100_I2C_ADDR
-# define HAL_COMPASS_RM3100_I2C_ADDR 0x20
+# define HAL_COMPASS_RM3100_I2C_ADDR1 0x20
+# define HAL_COMPASS_RM3100_I2C_ADDR2 0x21
```

Comment:
These all overlap with the SDP3X, but it does look like they both have enough probing I don't think it's a huge risk.

Reply:
yes. Unfortunately the SDP3X driver does writes before first reads, but at least it isn't probed for unless you set the type